apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/QueryLoggingListener.java

```diff
@@ -118,66 +118,64 @@ public void beforeQuery(ExecutionInfo execInfo, List<QueryInfo> queryInfoList) {
     BENE_BY_ID_OMIT_IDENTIFIERS(
         "bene_by_id.omit_hicns_and_mbis",
         (s ->
-            s.contains("from \"Beneficiaries\"")
-                && s.contains("\"beneficiaryId\"=")
+            s.contains(" from beneficiaries ")
+                && s.contains("bene_id=")
                 && !s.contains(" join ")
-                && !s.contains("\"hicn\"="))),
+                && !s.contains("bene_crnt_hic_num="))),
 
     BENE_BY_ID_INCLUDE_IDENTIFIERS(
         "bene_by_id.include_hicns_and_mbis",
         (s ->
-            s.contains("from \"Beneficiaries\"")
-                && s.contains("\"beneficiaryId\"=")
+            s.contains(" from beneficiaries ")
+                && s.contains("bene_id=")
                 && s.contains(" join ")
-                && !s.contains("\"hicn\"="))),
+                && !s.contains("bene_crnt_hic_num="))),
 
     BENE_BY_HICN_HISTORY(
         "bene_by_hicn.hicns_from_beneficiarieshistory",
-        (s -> s.contains(" from \"BeneficiariesHistory\" "))),
+        (s -> s.contains(" from beneficiaries_history ") && s.contains("bene_crnt_hic_num="))),
```

Comment:
I'll check, although it certainly doesn't hurt things, and in fact a case could be made that since we are checking beneficiary_history via a hicn, that we should have actually checked that the query includes the hicn; the prev test would have passed if we passed in an MBI for example.